sympy/physics/optics/tests/test_utils.py

```diff
@@ -81,11 +93,17 @@ def test_deviation():
 
 
 def test_brewster_angle():
-    m1 = Medium('m1', permittivity=e0, n=1)
-    m2 = Medium('m2', permittivity=e0, n=1.33)
```

Comment:
Maybe keep this test as well? That is, just duplicate the assert after these two lines.